Packs/Cyberwatch/Integrations/Cyberwatch/Cyberwatch.py

```diff
@@ -464,6 +464,199 @@ def get_one_asset(self, params, namespace="vulnerabilities"):
+            paginate_total_results = int(raw_response.headers.get("x-total", len(response)))
+            while curr_page * paginate_objperpage < paginate_total_results:
+                curr_page += 1
+                params["page"] = curr_page
+                raw_response = self._http_request(
+                    method="GET",
+                    url_suffix=path,
+                    resp_type="response",
+                    params=params,
+                )
+                response += raw_response.json()
+
+        return response
+
+    def get_sysadmin_one_asset(self, params):
+        """
+        Retrieve a single server from the /assets/servers/<ID> route (Sysadmin view).
+        Expects: params['id'] (int).
+        """
+        if "id" not in params:
+            raise DemistoException("Please provide an asset ID")
+
+        path = "/api/v3/assets/servers/" + str(params["id"])
+        response = self._http_request(method="GET", url_suffix=path, params=params)
+        return response
+
+    def get_compliance_assets(self, params):
+        """
+        Retrieve assets from the /compliance/assets route (Compliance view).
+        Behaviour mirrors get_assets() but returns native /compliance payloads.
+        """
+        path = "/api/v3/compliance/assets"
+
+        # if a page is given in params, we will fetch only the provided page
+        if "page" in params:
+            response = self._http_request(method="GET", url_suffix=path, params=params)
+        # if no page is given, we will fetch all the assets
+        else:
+            demisto.info('Fetching all Compliance Assets by default. You can override this by specifying the "page" parameter')
+            response = []
+            curr_page = 1
+            params["page"] = curr_page
+            raw_response = self._http_request(
+                method="GET",
+                url_suffix=path,
+                resp_type="response",
+                params=params,
+            )
+            response += raw_response.json()
+            paginate_objperpage = int(raw_response.headers.get("x-per-page", len(response) or 1))
+            paginate_total_results = int(raw_response.headers.get("x-total", len(response)))
+            while curr_page * paginate_objperpage < paginate_total_results:
+                curr_page += 1
+                params["page"] = curr_page
+                raw_response = self._http_request(
+                    method="GET",
+                    url_suffix=path,
+                    resp_type="response",
+                    params=params,
+                )
+                response += raw_response.json()
+
+        return response
+
+    def get_compliance_one_asset(self, params):
+        """
+        Retrieve a single asset from the /compliance/servers/<ID> route (Compliance view).
+        Expects: params['id'] (int).
+        Returns:
+        {
```

Comment:
why did you add a real data example to the doc? no need to give an example, return is for the type of data returned